A fix to https://bugzilla.mozilla.org/show_bug.cgi?id=1232285, which introduced division of crashAfterMS by PR_MillisecondsToInterval(1000). crashAfterMS has to be divided by 1000 to get number of seconds.
Will not send upstream.

--- a/toolkit/components/terminator/nsTerminator.cpp	2017-03-29 11:41:02.600935223 +0800
+++ b/toolkit/components/terminator/nsTerminator.cpp	2017-03-29 11:41:34.158337115 +0800
@@ -388,8 +388,7 @@
   }
 
   UniquePtr<Options> options(new Options());
-  const PRIntervalTime ticksDuration = PR_MillisecondsToInterval(1000);
-  options->crashAfterTicks = crashAfterMS / ticksDuration;
+  options->crashAfterTicks = crashAfterMS / 1000;
 
   DebugOnly<PRThread*> watchdogThread = CreateSystemThread(RunWatchdog,
                                                 options.release());
